build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -324,6 +336,16 @@ require.config({
     waitSeconds: 0,
     paths: customModulePaths
 });
+
+requirejs.onResourceLoad = function (context, map, depArray) {
+  // TODO Handle dynamic changes in module dependancies
```

Comment:
This may already be handling this (if this function is invoked whenever new modules are loaded)?